basculehttp/constructor.go

```diff
@@ -103,7 +109,8 @@ func (c *constructor) decorate(next http.Handler) http.Handler {
 		authorization := request.Header.Get(c.headerName)
 		if len(authorization) == 0 {
 			c.error(logger, MissingHeader, "", errNoAuthHeader)
-			response.WriteHeader(http.StatusForbidden)
+			response.Header().Set("WWW-Authenticate", string(c.unauthorizedTokenType))
+			response.WriteHeader(http.StatusUnauthorized)
```

Comment:
This is definitely the correct header we should be returning.  The reason we return `StatusForbidden` everywhere is for backwards compatibility with the old `secure` package that was once used to provide our authorization middleware in XMiDT services.  I think it would make a lot of sense to have this be another Option, so that new services can follow the spec but old services can switch over at a desirable time (since this is an API change).